tests/auth/test_generic_auth.py

```diff
@@ -115,99 +95,33 @@ def gethostbyname_patch(x):
     monkeypatch.setattr(socket, "gethostbyname", gethostbyname_patch)
 
     # A valid update
-    updated_settings = copy.deepcopy(settings)
-    updated_settings["settings"]["imap_server_host"] = "mail.office365.com"
-    updated_settings["settings"]["smtp_server_host"] = "mail.office365.com"
-    updated_settings["settings"]["name"] = "Neu!"
-    account = handler.update_account(account, updated_settings["settings"])
+    updated_data = attr.evolve(
```

Comment:
I saw. I didn't feel like it needed a convenience method. I think I prefer doing it explicitly via `attr` in tests since it's not really meant to evolve apart from the test cases.